<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>material_info</class>
 <widget class="QDialog" name="material_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>存货修改/删除/导出Excel</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>1021</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1021</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>筛选条件</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1001</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>存货编码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>存货名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>规格型号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="spec_lineEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>库存位置：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="position_lineEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>801</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>结存数量范围(含):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="st_num">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>——</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ed_num">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string> 单价(元)范围(含):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="st_per_price">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>——</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ed_per_price">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="query_btn">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>筛选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_query_info">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>清除筛选</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1001</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;1.&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;默认&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#000000;&quot;&gt;列出&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;所有存货&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#000000;&quot;&gt;。&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;2.存货编码&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;精确匹配&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;，其余项支持&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;模糊搜索&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#000000;&quot;&gt;。&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;3.多个输入用&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;%&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;隔开，例如：ID1%ID2%ID3。4.某项为&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;空&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;表示该项&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;不做筛选&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#000000;&quot;&gt;。5.&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;ctrl键&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#000000;&quot;&gt;可同时选中多行并删除，但修改时只能选中1行&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="update_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>修改选中存货</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_excel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>550</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>导出表中数据至Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_material_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>删除选中存货</string>
   </property>
  </widget>
  <widget class="QPushButton" name="total_data_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>汇总统计</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
